### PR TITLE
(maint) bump travis/appveyor lth version to 0.7

### PR DESCRIPTION
**Pull request by branan**

### CODE REVIEW

File: lib/CMakeLists.txt [hunk 1]
```diff
@@ -302,6 +302,7 @@ target_link_libraries(libfacter PRIVATE
     ${CMAKE_THREAD_LIBS_INIT}
 )
 
+symbol_exports(libfactersrc "${CMAKE_CURRENT_LIST_DIR}/inc/facter/export.h")
```
Comment (MikaelSmith) [inline lib/CMakeLists.txt]: Ah, does Leatherman need to change for this to work?

Comment (branan) [inline lib/CMakeLists.txt]: Gonna find out as soon as appveyor hits it

Comment (branan) [inline lib/CMakeLists.txt]: I'm also not *entirely* certain that's the cause of our problem, but it seems to be something that approximates "GCC is assuming declarations are dllimport, but definitions aren't". I actually can't find `dllimport` or `dllexport` anywhere in facter or leatherman, so I have no idea how our export stuff works on Windows.

**Comment by branan** [comment]
@MikaelSmith I've had no luck trying to add the export helper macro to other targets to get this running on Windows. Adding it to just the `libfactersrc` target resulted in the same error as before, and it doesn't work to add it to `libfacter_test` because it only works on library targets. Do you have any thoughts?

File: lib/tests/CMakeLists.txt [hunk 2]
```diff
@@ -126,7 +126,9 @@ target_link_libraries(libfacter_test
     ${LEATHERMAN_LIBRARIES}
     ${CURL_LIBRARIES}
     ${CMAKE_THREAD_LIBS_INIT}
-)
+    )
```
Comment (MikaelSmith) [inline lib/tests/CMakeLists.txt]: No longer matches other formatting...

Comment (branan) [inline lib/tests/CMakeLists.txt]: I'm blame my editor. Fixed.

**Comment by MikaelSmith** [comment]
👍

**Pull request merged**